### ISSUE TITLE
Unable to upgrade from 0.2.0 to 0.3.0

### ISSUE THREAD

**Issue by cb341** [BaseSecrete/active_hashcash#3]
I encountered an issue while upgrading from active_hashcash version 0.2.0 to version 0.3.0. I am unable to execute the migrations successfully.

Steps to reproduce:
1. Create a new Rails project:
```
rails new hashcash-upgrade
```
2. Install `active_hashcash` gem version 0.3.0:
```
gem install active_hashcash -v 0.3.0
```

3. Attempt to run migrations:
```
bundle exec rails active_hashcash:install:migrations
```

**Error message:**
```
rails aborted!
Don't know how to build task 'active_hashcash:install:migrations' (See the list of available tasks with `rails --tasks`)
Did you mean?  active_storage:install:migrations
               action_mailbox:install:migrations
```

**Comment by alexisbernard** [comment]
Thanks for the report. I forgot to include the db directory into the gemspec ... Curiously, there is no issue when the gem is installed via a local path or a Git URL. I published the version 0.3.1.

**Comment by cb341** [comment]
I really appreciate your help with this! I updated the gem to version 0.3.1 and the migrations went through without a problem. Thanks a lot for your support!
